packages/opentelemetry-metrics/src/export/aggregators/histogram.ts

```diff
@@ -0,0 +1,96 @@
+/*!
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Aggregator, Point } from '../types';
+import { HrTime } from '@opentelemetry/api';
+import { hrTime } from '@opentelemetry/core';
+
+export type Checkpoint = {
```

Comment:
I would prefer to call this `Histogram` instead of `Checkpoint`, any specific reason for this name?

Reply:
I've tried to expose the same API that Go does ([here](https://github.com/open-telemetry/opentelemetry-go/blob/master/sdk/metric/aggregator/histogram/histogram.go#L135)) but you are right, `Histogram` make more sense